Satyam1923/Spring-Music-Player
# Added Dockerfile

### Description
Created a dockerfile for both frontend and backend and updated the docs

- Placeholder: "Fixes #157 "

### Changes

### Testing Instructions
Detailed instructions on how to test the changes. Include any setup needed and specific test cases.
1. Pull this branch.
2. cd frontend || backend 
3. Run `docker build -t frontend:v1 . ` to create a docker image that will have frontend code.  // It is a one time process 
4. Run `docker run -p 3000:3000 frontend:v1` to run the docker image.
5. Verify that ...


### Checklist
Make sure to check off all the items before submitting. Mark with [x] if done.
